client/components/site-selector/add-site.jsx

```diff
@@ -34,4 +34,9 @@ class SiteSelectorAddSite extends Component {
 	}
 }
 
-export default connect( null, { recordTracksEvent } )( localize( SiteSelectorAddSite ) );
+export default connect(
+	( state ) => ( {
+		onboardingUrl: getOnboardingUrl( state ),
```

Comment:
The site selector is used in Jetpack Cloud, too. I can see we haven't taken the Jetpack Cloud environments into account - has this been considered? Will this change have any unexpected effects there? 

cc @Automattic/jetpack-voyager for their Jetpack Cloud expertise.

Reply:
FWIW I've pinged the Voyager team in Slack about this: p1608201558180500-slack-CQXNTE9K9

Reply:
Thanks for bringing this up Marin. I can see two broken flows in Jetpack Cloud:

1) User with no sites.
![CloudNoSites](https://user-images.githubusercontent.com/3418513/102494227-1abd2a00-4053-11eb-9e4e-3b3817b78d23.gif)

2) User with sites wants to add a new site.
![CloudSwitchSite](https://user-images.githubusercontent.com/3418513/102494225-198bfd00-4053-11eb-81e6-41687410cc2e.gif)

However, these two issues were already present in production and were not introduced by these changes. I think the sensible thing to do is to create a card in our Maintainance board and fix them in a follow-up PR.

Would you all be okay with that?

Reply:
Thanks for chiming in so quickly, @rcanepa. 

As far as this PR is concerned, it seems like there is no regression introduced on the Jetpack Cloud side, so it should be good to go from this side @razvanpapadopol.

And I appreciate you taking the time to report and fix this issue separately. Cross-team collab FTW!